### PR TITLE
Okta CLI with Fig

### PR DESCRIPTION
**Pull request by alisaduncan**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [x] A title that's approved by Dev Advocacy
- [x] A URL that's approved by Dev Advocacy
- [x] The content has been run through Grammarly (https://www.grammarly.com/)
- [x] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2022-01-24-okta-cli-completion.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+### Managing Okta apps
+
+You may need to create Okta apps and get app configurations. When you need to create an Okta app, the Okta CLI is very helpful as it steps you through the process. However, if you are an Okta power user, you may be fine creating the Okta app in one shot by passing in the app settings. The Okta CLI autocompletion is pretty handy here as it shows you all the options you can pass in!
+
+{% img blog/okta-cli-completion/create-apps.jpg alt:"Optional flags to pass in while creating an Okta app" width:"800" %}{: .center-image }
+
+### Help is on the way!
+
+Getting more detailed help information is easier too! The autocompletion context menu helps you navigate to the help output for a command. And yup, you can get help about `help` command. Pretty meta, right?
```
Comment (bdemers) [inline _source/_posts/2022-01-24-okta-cli-completion.md]: ❤️

File: _source/_posts/2022-01-24-okta-cli-completion.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+layout: blog_post
+title: "Boost your productivity using Okta CLI and Fig"
+author: alisa-duncan
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Introducing Okta CLI autocompletion in Fig to power up your commandline"
+tags: [security, oidc, cli]
+tweets:
+- "Looking to power up your commandline? 💪 Use Okta CLI with @fig for ultimate productivity."
```
Comment (havi) [inline _source/_posts/2022-01-24-okta-cli-completion.md]:
```suggestion
- "Looking to power up your command line? 💪  Use Okta CLI with @fig for ultimate productivity."
```
added space

File: _source/_posts/2022-01-24-okta-cli-completion.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- ""
+image: blog/okta-cli-completion/okta-cli-completions-social.jpg
+type: awareness
+---
+
+CLIs are great. I love the speed and productivity increases I get when using a CLI, but memorizing commands, when commands need arguments, options, flags, and so on, can be daunting. Luckily, there are tools available for CLI fans out there, and one tool I've been enjoying is [Fig](https://fig.io/).
+
+## Fig powers up your CLI productivity
+
+Fig add autocompletion to supported terminals, which makes using CLIs so much easier. Using Git? You'll see a small window pop up with different commands and options.
```
Comment (havi) [inline _source/_posts/2022-01-24-okta-cli-completion.md]:
```suggestion
Fig adds autocompletion to supported terminals, which makes using CLIs so much easier. Using Git? You'll see a small window pop up with different commands and options. 
```
typo

File: _source/_posts/2022-01-24-okta-cli-completion.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+tags: [security, oidc, cli]
+tweets:
+- "Looking to power up your command line? 💪  Use Okta CLI with @fig for ultimate productivity."
+- "Too many terminal commands to remember? Get inline Okta CLI help with @fig! 🎉"
+- ""
+image: blog/okta-cli-completion/okta-cli-completions-social.jpg
+type: awareness
+---
+
+CLIs are great. I love the speed and productivity increases I get when using a CLI, but memorizing commands, when commands need arguments, options, flags, and so on, can be daunting. Luckily, there are tools available for CLI fans out there, and one tool I've been enjoying is [Fig](https://fig.io/).
```
Comment (havi) [inline _source/_posts/2022-01-24-okta-cli-completion.md]:
```suggestion
CLIs are great. I love the speed and productivity increases I get when using a CLI, but memorizing commands – especially when commands need arguments, options, flags, and so on – can be daunting. Luckily, there are tools available for CLI fans out there, and one tool I've been enjoying is [Fig](https://fig.io/).
```
punctuation tweak to ease readability

File: _source/_posts/2022-01-24-okta-cli-completion.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+{% img blog/okta-cli-completion/git-commands.jpg alt:"Git command autocompletions in Fig" width:"800" %}{: .center-image }
+
+It's a cool way to reinforce remembering Git command-line operations. But most of us use Git on a daily basis, so we probably have a good command... of our Git commands. 😳 Hilarious pun aside, the branch names in the context menu are pretty darn helpful.
+
+## Boosting the power of Okta CLI
+
+How about when it's a CLI that you might not use daily? This is where the autocompletion shines and really helps you power up!
+
+We're pleased to announce that [Okta CLI](https://cli.okta.com/) autocompletion are available in Fig!  When you use a supported terminal, you'll see hints to help you navigate through the commands and flags, so managing your Okta applications have never been easier.
```
Comment (havi) [inline _source/_posts/2022-01-24-okta-cli-completion.md]:
```suggestion
We're pleased to announce that [Okta CLI](https://cli.okta.com/) autocompletion is available in Fig!  When you use a supported terminal, you'll see hints to help you navigate through the commands and flags, so managing your Okta applications has never been easier.
```
grammar / usage tweaks

File: _source/_posts/2022-01-24-okta-cli-completion.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+{% img blog/okta-cli-completion/all-commands.jpg alt:"All command options in Okta CLI" width:"800" %}{: .center-image }
+
+The autocompletion context menu lists all the commands available and shows you which command requires an argument. The `okta start` command allows you to pass in a name, and you can see the optional argument at a glance.
```
Comment (havi) [inline _source/_posts/2022-01-24-okta-cli-completion.md]:
```suggestion
The autocomplete context menu lists all the commands available and shows you which command requires an argument. The `okta start` command allows you to pass in a name, and you can see the optional argument at a glance.
```
using autocomplete to modify the noun "menu"

File: _source/_posts/2022-01-24-okta-cli-completion.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+### Managing Okta apps
+
+You may need to create Okta apps and get app configurations. When you need to create an Okta app, the Okta CLI is very helpful as it steps you through the process. However, if you are an Okta power user, you may be fine creating the Okta app in one shot by passing in the app settings. The Okta CLI autocompletion is pretty handy here as it shows you all the options you can pass in!
```
Comment (havi) [inline _source/_posts/2022-01-24-okta-cli-completion.md]:
```suggestion
You may need to create Okta apps and get app configurations. When you need to create an Okta app, the Okta CLI helps you step through the process. However, if you are an Okta power user, you may be fine creating the Okta app in one shot by passing in the app settings. In this case, the Okta CLI autocomplete is pretty handy as it shows you all the options you can pass in!
```
tweaks for terseness & clarity

File: _source/_posts/2022-01-24-okta-cli-completion.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+{% img blog/okta-cli-completion/create-apps.jpg alt:"Optional flags to pass in while creating an Okta app" width:"800" %}{: .center-image }
+
+### Help is on the way!
+
+Getting more detailed help information is easier too! The autocompletion context menu helps you navigate to the help output for a command. And yup, you can get help about the `help` command. Pretty meta, right?
```
Comment (havi) [inline _source/_posts/2022-01-24-okta-cli-completion.md]:
```suggestion
Getting more detailed help information is easier too! The autocomplete context menu helps you navigate to the help output for a command. And yup, you can get help about the `help` command. Pretty meta, right?
```

File: _source/_posts/2022-01-24-okta-cli-completion.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+{% img blog/okta-cli-completion/help-commands.jpg alt:"Okta CLI help context" width:"800" %}{: .center-image }
+
+## Supported terminals for Okta CLI using Fig
+
+Now that you're excited, here's the fine print. Fig currently only supports macOS. 😰  And only certain terminals and shells within macOS at that. Read [Fig's FAQ](https://fig.io/support/other/faq) for supported terminals and shells, as well as information for work support cross-platform and more terminal/shell integrations. Don't forget to get on the waiting list for your OS or shell, or submit an issue so Fig knows how to prioritize future work.
+
+## Learn more
+
+I hope the autocompletion helps you! I'd love to hear how you use Okta CLI and what tools you use to power up your terminal commands. We welcome issues and PRs in the [okta-cli GitHub repo](https://github.com/okta/okta-cli) too.
```
Comment (havi) [inline _source/_posts/2022-01-24-okta-cli-completion.md]:
```suggestion
I hope this autocomplete feature helps you! I'd love to hear how you use Okta CLI and what tools you use to power up your terminal commands. We welcome issues and PRs in the [okta-cli GitHub repo](https://github.com/okta/okta-cli) too.
```

**Review by havi — CHANGES_REQUESTED** [review]
small edits, sweet post.

**Review by havi — APPROVED** [review]
let's ship it!

**Pull request merged**